releasenotes/notes/27049.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: feature
+area: installation
+
+releaseNotes:
+- |
+  **Improved** Enable configurng multi-network without using [meshNetworks](https://istio.io/latest/docs/reference/config/istio.mesh.v1alpha1/#MeshNetworks).
```

Comment:
```suggestion
  **Added** the ability to use `topology.istio.io/network` and `topology.istio.io/gatewayPort` on the gateway's Kubernetes Service rather than [meshNetworks](https://istio.io/latest/docs/reference/config/istio.mesh.v1alpha1/#MeshNetworks) to mark a service as a gateway. 
```